### ISSUE TITLE
no possibility to set network --activate

### ISSUE THREAD

**Issue by jnovy** [quattor/aii#146]
Hi,

we are using image based builds which in kickstart look like:

liveimg --url=myimage

on RHEL7. The problem is that when anaconda switches to stage2 then it doesn't reactivate network interface leading to image download failure.

Replacing:
network --bootproto=dhcp

with

network --bootproto=dhcp --activate

fixes it.

There is however to my knowledge no way how to force aii-ks to add the --activate option or is it?

Thanks,
Jindrich

**Comment by stdweird** [comment]
should be an easy fix. is `--activate` and EL7 only option? (and out of interest, why not use static network config in ks/anaconda?)

**Comment by jnovy** [comment]
yep, EL7 only. There is even a possibility to do multiple network statements in ks file like:

network --bootproto=dhcp
...
network --activate

which EL7 anaconda is able to understand and it would make the aii-ks code more readable (just addition).

WRT static net configuration - we need it as described above due to our template side conf/and hw setup.

**Comment by jnovy** [comment]
Also it's worth noting that in case of ordinary "url --url" package installations it's not needed. It's required for image deployments only.
